qiskit_machine_learning/kernels/algorithms/quantum_kernel_trainer.py

```diff
@@ -88,7 +89,7 @@ class QuantumKernelTrainer:
 
     def __init__(
         self,
-        quantum_kernel: QuantumKernel,
+        quantum_kernel: TrainableKernel,
```

Comment:
The example code above ideally should be updated to use a newer primitive based kernel

Reply:
The code snippet is updated.